test/WebJobs.Script.Tests/Scale/FunctionsScaleMonitorServiceTests.cs

```diff
@@ -192,6 +204,25 @@ public async Task OnTimer_MonitorFailuresAreHandled()
             var metricsWritten = _metricsRepository.Metrics[monitor2].Take(5);
             Assert.Equal(testMetrics2, metricsWritten);
         }
+
+        [Fact]
+        public async Task TakeMetricsSamplesAsync_Skips_TakingMertrics()
```

Comment:
spelling

Reply:
We testing that if there a target scaler in the extension we do not do sampling.